src/upload/catalog/controller/extension/payment/libwebpay_rest/HealthCheck.php

```diff
@@ -24,97 +21,71 @@ public function __construct($config) {
-                }
-            }
-            $status = 'OK';
-            $result = array(
-                'status' => $status,
+            $version = $extension === 'openssl' ? OPENSSL_VERSION_TEXT : phpversion($extension) ?? 'PHP Extension Compiled. ver:' . phpversion();
+            return [
+                'status' => 'OK',
                 'version' => $version
-            );
-        } else {
-            $result = array(
-                'status' => 'Error!',
-                'version' => 'No Disponible'
-            );
+            ];
         }
-        return $result;
+        return [
+            'status' => 'Error!',
+            'version' => 'No Disponible'
+        ];
+
     }
 
     // obtiene ultimas versiones
     // obtiene versiones ultima publica en github (no compatible con virtuemart) lo ideal es que el :usuario/:repo sean entregados como string
     // permite un maximo de 60 consultas por hora
-    private function getLastGitHubReleaseVersion($string){
+    private function getLastGitHubReleaseVersion($string): string{
         $baseurl = 'https://api.github.com/repos/'.$string.'/releases/latest';
         $agent = 'Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1; SV1)';
         $ch = curl_init();
         curl_setopt($ch,CURLOPT_URL,$baseurl);
         curl_setopt($ch,CURLOPT_RETURNTRANSFER,true);
         curl_setopt($ch, CURLOPT_USERAGENT, $agent);
-        //curl_setopt($ch,CURLOPT_HEADER, false);
         $content=curl_exec($ch);
         curl_close($ch);
         $con = json_decode($content, true);
         $version = array_key_exists('tag_name',$con) ? $con['tag_name'] : '';
-        return $version;
+        return $con['tag_name'] ?? '';
     }
 
     // funcion para obtener info de cada ecommerce, si el ecommerce es incorrecto o no esta seteado se escapa como respuesta "NO APLICA"
-    private function getEcommerceInfo($ecommerce){
-        $actualversion = TransbankSdkWebpay::PLUGIN_VERSION;
-        $lastversion = $this->getLastGitHubReleaseVersion('opencart/opencart');
-        $currentplugin = TransbankSdkWebpay::PLUGIN_VERSION;
-        $result = array(
-            'current_ecommerce_version' => $actualversion,
-            'last_ecommerce_version' => $lastversion,
-            'current_plugin_version' => $currentplugin
-        );
-        return $result;
+    private function getEcommerceInfo(): array{
+        return [
+            'current_ecommerce_version' => TransbankSdkWebpay::PLUGIN_VERSION,
+            'last_ecommerce_version' => $this->getLastGitHubReleaseVersion('opencart/opencart'),
+            'current_plugin_version' => TransbankSdkWebpay::PLUGIN_VERSION
+        ];
     }
 
     // creacion de retornos
     // arma array que entrega informacion del ecommerce: nombre, version instalada, ultima version disponible
-    private function getPluginInfo($ecommerce){
+    private function getPluginInfo($ecommerce): array {
         $data = $this->getEcommerceInfo($ecommerce);
```

Comment:
Above you modified this function so that it now does not receive parameters, but here you are still sending parameters
```suggestion
        $data = $this->getEcommerceInfo();
```